### PR TITLE
feat: Filter symbols in the frontend

### PR DESCRIPTION
**Pull request by pedrohusky**
## Description

A simple search box to filter the symbols. It is animated and such.

## Related Issue

Filter symbols in the frontend - [#120]

## Motivation and Context

Sometimes is hard to find the symbol with many monitored. This will help us.

## How Has This Been Tested?

Searching something, then the data was filtered.

## Screenshots:

Captura de Tela (153)
Captura de Tela (154)

### CODE REVIEW

File: public/js/SearchIcon.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    searchName = value.toUpperCase();
+
+    this.setState({
+      searchName
+    });
+    this.props.searchSymbolWithName(searchName);
+  }
+
+  render() {
+    document.addEventListener("touchstart", function () { }, true);
```
Comment (chrisleekr) [inline public/js/SearchIcon.js:36]: What does this line do?

**Comment by chrisleekr** [comment]
Nice work @pedrohusky 

I think you still have some formatting issues, but I can get it to fix.

Let me take a look when I get time.

File: public/js/SearchIcon.js (hunk 1 above)
Comment (pedrohusky) [reply public/js/SearchIcon.js:36]: I used that code from the search box that I found. I didn't try without it though. I think it sets the event listener to the search box? But from what I know it should work without it. Let me test :)

Comment (chrisleekr) [reply public/js/SearchIcon.js:36]: Yeah, I don't think that line is necessary, but yeah, test it out. :)

**Comment by chrisleekr** [comment]
Hmm, your code was ok and I tried to make it better, but I found some browser compatibility issues. 

I may focus on another feature and get back to this.

**Comment by pedrohusky** [comment]
> Hmm, your code was ok and I tried to make it better, but I found some browser compatibility issues. 
> 
> I may focus on another feature and get back to this. 

Hi. What issues? Just to know.

@chrisleekr please see my last comment in security screen pull request. It is urgent.

**Comment by chrisleekr** [comment]
@pedrohusky I close this PR because I made another PR #253 with the latest code.
Thanks again for your contributions!

**Pull request closed without merging**